src/dialog.tsx

```diff
@@ -312,34 +362,19 @@ const ModalDialog = (props: ModalDialogProps) => {
         document.addEventListener('keydown', handleKeyDown);
     }, [handleKeyDown]);
 
-    useNativeDialog({
-        renderNative,
-        acceptText: props.acceptText || context.texts.dialogAcceptButton,
-        cancelText: props.cancelText || context.texts.dialogCancelButton,
-        showCancel: props.showCancel,
-        message: props.message,
-        title: props.title,
-        onAccept: props.onAccept,
-        onCancel: props.onCancel,
-    });
-
     React.useEffect(() => {
         if (!animationsSupported(platformOverrides)) {
             addKeyDownListener();
         }
 
-        if (
-            (renderNative || !animationsSupported(platformOverrides)) &&
-            props.isClosing &&
-            props.onCloseTransitionEnd
-        ) {
+        if (!animationsSupported(platformOverrides) && props.isClosing && props.onCloseTransitionEnd) {
```

Comment:
why this change?

Reply:
just move the `renderNative` to new component, but as renderNative is not needed there, this could be restored as it was too